src/python/arcor2_arserver/user.py

```diff
@@ -8,11 +8,19 @@
 class Users:
     def __init__(self) -> None:
 
+        self._interfaces: Set[WsClient] = set()
         self._users_ui: Dict[str, int] = {}
```

Comment:
Spíš jen pro zajímavost - na takovéto případy se hodí https://github.com/jab/bidict.

Reply:
Jestli dobře koukám, tak z `self._users_ui` a `self._ui_users` se nikde nemaže...

Reply:
Nemaže, taky jsem nad tím uvažoval. Je to potřeba? Asi jen v případě, že se na jeden server budou připojovat stovky uživatelů, tak by to mohlo konzumovat více paměti... 
Ale asi to můžu promazat s timeoutem zámku, nic to nestojí...

Reply:
Nebo to možná s timeoutem ani nesouvisí a může se to pročistit s logoutem... ještě kouknu

Reply:
No, tak teoreticky to asi potřeba není, ale přišlo by mi takový čistější mít tam jen aktuální záznamy.

Reply:
Třeba budeme chtít do budoucna udělat RPC na vylistování dalších uživatelů, nebo tak něco...